src/main/java/io/kestra/plugin/azure/datafactory/CreateRun.java

```diff
@@ -0,0 +1,203 @@
+        Logger logger = runContext.logger();
+
+        //Authentication
+        DataFactoryManager manager = this.getDataFactoryManager(runContext);
+        logger.info("Successfully authenticate to Azure Data Factory");
+
+        //Create running pipeline
+        final String resourceGroupName = this.resourceGroupName.as(runContext, String.class);
+        final String factoryName = this.factoryName.as(runContext, String.class);
+        final String pipelineName = this.pipelineName.as(runContext, String.class);
+
+        final var pipelineRunResponse = manager.pipelines()
+                .createRunWithResponse(resourceGroupName,
+                        factoryName,
+                        pipelineName,
+                        null,
+                        null,
+                        null,
+                        null,
+                        parameters.asMap(runContext, String.class, Object.class),
+                        Context.NONE
+                );
+
+        if(pipelineRunResponse.getStatusCode() != HttpResponseStatus.OK.code()) {
+            throw new RuntimeException();
+        }
+        logger.info("Created run for pipeline '{}'", pipelineName);
+
+        // Get running pipeline
+        final String runId = pipelineRunResponse.getValue().runId();
+
+        PipelineRun pipelineRun = getRunningPipeline(resourceGroupName,factoryName, runId, manager);
+        String runStatus = pipelineRun.status();
+
+        while ("InProgress".equals(runStatus)) {
+            Thread.sleep(1000);
+            pipelineRun = getRunningPipeline(resourceGroupName,factoryName, runId, manager);;
+            runStatus = pipelineRun.status();
+        }
+
+        runContext.metric(Timer.of("pipeline.duration.ms", Duration.of(pipelineRun.durationInMs(), ChronoUnit.MILLIS)));
+
+        if(runStatus.equals(PIPELINE_SUCCEEDED_STATUS)) {
+            logger.info("Pipeline '{}' with runId '{} finished with status '{}'", pipelineName, runId, runStatus);
+        } else {
+            logger.error("Pipeline '{}' with runId '{} finished with status '{}'", pipelineName, runId, runStatus);
+        }
+        logger.info("Pipeline run message : {}", pipelineRun.message());
+
+        final var activitiesResponse = manager.activityRuns().queryByPipelineRunWithResponse(
+                resourceGroupName,
+                factoryName,
+                runId,
+                new RunFilterParameters()
+                        .withLastUpdatedAfter(pipelineRun.runStart())
+                        .withLastUpdatedBefore(pipelineRun.runEnd()),
+                com.azure.core.util.Context.NONE);
+
+        if(activitiesResponse.getStatusCode() != HttpResponseStatus.OK.code()) {
+            throw new RuntimeException();
+        }
+        logger.info("Logging pipeline activities");
+        ObjectMapper jsonMapper = JacksonMapper.ofJson();
+        jsonMapper.setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY);
```

Comment:
When I don't specify `jsonMapper.setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY);` I get this error and I am unable to parse : 
`No serializer found for class com.azure.resourcemanager.datafactory.models.ActivityRun and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS)`

Reply:
Removed